test/unit/auditingTest.js

```diff
@@ -301,7 +301,7 @@ describe('Auditing', () => {
 
     after(async () => {
       config.auditing.auditEvents = JSON.parse(_restore)
-      await Promise.all(servers.map(s => s.close()))
+      await servers.map(s => s.close())
```

Comment:
What is this line supposed to do now? Why the `.map` and `await`?

Reply:
Hey Brett, so I am still investigating, havent made much headway as to why they fail. The tests only pass when the close function is not in a promise. The program times out before the close is event is fired. I will push something that's a bit cleaner just now

Reply:
> What is this line supposed to do now? Why the `.map` and `await`?

the line is for closing the servers that are in an array